apps/new-dealer/views/partials/weapons-ammo-unspecified.html

```diff
@@ -1,7 +1,5 @@
-<div id="{{toggle}}-panel" class="reveal" aria-hidden="false">
-  <div class='panel-indent'>
+<div id="{{toggle}}-panel" class="govuk-checkboxes__conditional govuk-checkboxes__conditional--hidden" aria-hidden="false">
     {{#textarea}}{{toggle}}{{/textarea}}
-  </div>
 </div>
 <div class="form-block">
```

Comment:
Should form-block be removed or replaced?

Reply:
I have replaced this with “govuk-checkboxes__divider” per govuk design system.